runtime/userspace/syscall/src/dma.rs

```diff
@@ -154,3 +154,9 @@ mod dma_ro_buffer {
 mod dma_subscribe {
     pub const XFER_DONE: u32 = 0;
 }
+
+pub fn mcu_sram_to_axi_address(addr: u32) -> u64 {
+    const MCU_SRAM_HI_OFFSET: u64 = 0x1000_0000;
```

Comment:
Not in this PR necessarily, but we don't want to hardcode these addresses in a generic `runtime/` component -- this should be passed in from the platform-specific runtime.